include/torch-mlir/Dialect/Torch/IR/GeneratedTorchOps.td

```diff
@@ -5742,7 +5740,7 @@ def Torch_AtenScatterAddOp : Torch_Op<"aten.scatter_add", [
     HasValueSemantics,
     ReadOnly
   ]> {
-  let summary = "Generated op for `aten::scatter_add : (Tensor self, int dim, Tensor index, Tensor src) -> (Tensor)`";
+  let summary = "Generated op for `aten::scatter_add : (Tensor, int, Tensor, Tensor) -> (Tensor)`";
```

Comment:
You should rebase your local branch over the latest upstream torch-mlir, and then re-run the script `build_tools/update_torch_ods.sh`, these changes should be gone.